lib/Config/PrivacyPolicy.php

```diff
@@ -46,14 +46,14 @@ function getPrivacyPolicyContent() {
         WPFunctions::get()->__('MailPoet creates and stores two cookies if you are using WooCommerce and MailPoet together. Those cookies are:', 'mailpoet') .
         '</p>' .
         '<p>' .
-        WPFunctions::get()->__('Cookie name: mailpoet_revenue_tracking', 'mailpoet') .
+        WPFunctions::get()->sprintf( __('Cookie name: %s', 'mailpoet'), 'mailpoet_revenue_tracking' ) .
```

Comment:
This code doesn't work. There is no `sprintf` function on the `WPFunctions` object. This has to be the other way around: `sprintf(WPFunctions::get()->__('Cookie name: %s', 'mailpoet'), 'mailpoet_revenue_tracking')`.

Another problem is the coding standards. We don't use a space in the parentheses. This would break our build.

Reply:
Fixed.